commands/http/client.go

```diff
@@ -145,6 +158,18 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 	}
 }
 
+func streamingRequest(url string, req cmds.Request) (cmds.Response, error) {
+	conn, httpRes, err := websocket.DefaultDialer.Dial(url, nil)
+	if err != nil { return nil, err }
```

Comment:
expand this out so its not all on one line, i'm not a fan of dense code like this